docs/AboutUs.md

```diff
@@ -38,10 +38,10 @@ We are a team based in the [School of Computing, National University of Singapor
 
 ### Joshua Yip Sujun
 
-<img src="images/joshuayip.png" width="200px">
+<img src="images/joshy837.png" width="200px">
```

Comment:
His username is Joshy837 with capital J

Reply:
CS2103T convention is using lowercase for every type of github username in readme.md